content/docs/tasks/traffic-management/custom-gateway/index.md

```diff
@@ -0,0 +1,355 @@
+---
+title: Add a new Custom IngressGateway
+description: Describes how to add a new custom ingress gateway manually.
+weight: 89
+keywords: [ingress,traffic-management]
+---
+
+This post provide instructions to create manually a new custom ingress gateway with automatic provisioning of certificate based on cert-manager.
```

Comment:
Since this is now a task, the text must reflect the change:
This task provides ...
See my previous comment to fix the grammar issues.